apps/prairielearn/src/tests/fileEditor.test.ts

```diff
@@ -806,36 +764,29 @@ function doEdits(data) {
     // (C, C, C*, C*)
 
     editPost('save_and_sync', data.contentsB, data.url, true, false, null);
-    // (C, C, C*, C*)
-
-    waitForJobSequence(locals, 'Error');
-    editPost('pull_and_save_and_sync', data.contentsB, data.url, true, false, null);
-    // (B, B, C, B)
-
```

Comment:
As above, we now pull, which makes this succeed immediately without a separate "pull" step (which actually doesn't exist anymore).

Reply:
Agreed.